composer.json

```diff
@@ -75,6 +79,9 @@
 		"test": [
 			"@php ./vendor/bin/phpunit --no-coverage --order-by=random"
 		],
+	   	"static-analysis": [
+		  "@php ./vendor/bin/phpstan analyze src views wp-parsely.php uninstall.php -l 9 --memory-limit 536870912"
```

Comment:
May as well do `tests` as well.

Reply:
I excluded `tests` on purpose. There are some errors derived from PHPStan not finding the Yoast test helpers, which I want to address in the future. We can ignore those errors, but I think at this point they are just a distraction.

Reply:
Let's move the configuration to a `.neon` file ([example](https://github.com/google/web-stories-wp/pull/9816/files#diff-6f19df6a6307a48db0940e6897591fb08776d2db8a6134737aa708defdb5c92d)),

Reply:
I've moved the configuration to a `.neon` file and added tests, but I'm still trying to figure out how to make it work with Yoast.